lib/remove_bg/account.rb

```diff
@@ -0,0 +1,30 @@
+module RemoveBg
+  class Account
```

Comment:
Not sure if `AccountInfo` might be a better name for this (more verbose, but less likely to clash with any future features relating to account management). If so I'd also rename the call to fetch it from `RemoveBg.account` -> `RemoveBg.account_info`. What do you think @groe?

Reply:
Hmm yes good point. I think the more verbose version is better in this case